assets/js/src/newsletters/campaign_stats/newsletter_stats.jsx

```diff
@@ -47,9 +46,7 @@ const NewsletterGeneralStats = ({ newsletter }) => {
             headline={headlineClicked}
           />
         </div>
-        <RevenuesStats
-          revenue={newsletter.statistics.revenue}
-        />
+        {Hooks.applyFilters('mailpoet_newsletters_revenues_stats', null, newsletter.statistics.revenue)}
```

Comment:
What's the second (`null`) parameter for?

Reply:
When no `Hooks.addFilters()` is called (because in free there is none) that `null` parameter is what's being rendered. And since the free plugin has no revenue stats, nothing is rendered and only premium content is visible with a hook in the premium plugin.